### ISSUE TITLE
Qwak jump button

### ISSUE THREAD

**Issue by rashohyo** [amigavision/AmigaVision#147]
Is it possible to add jump button for qwak game?
thanks

**Comment by limi** [comment]
Yes, this should be possible by changing the configuration to prefer the CD32 version as default (I assume that one adds support). Will take a look before the next release.

**Comment by limi** [comment]
I don’t think the CD32 version made a difference, but give it a try, it should be using the CD32 config in the 2024.06.06 release. If that doesn’t fix it, we should file a bug upstream.

**Comment by limi** [comment]
CD32 version doesn’t seem to support button for jumping, so I filed an issue:

https://www.whdload.de/mantis/view.php?id=6548
